common/extensions/api/_api_features.json

```diff
@@ -61,17 +62,17 @@
   "sockets.tcp": {
     "dependencies": ["manifest:sockets"],
     "contexts": ["blessed_extension"],
-    "whitelist": ["3D9518A72EB02667A773B69DBA9E72E0F4A37423"]
```

Comment:
i assume 3D9518A72EB02667A773B69DBA9E72E0F4A37423 is webtorrent?

Reply:
yep